docs/src/user-guide/getting-started.md

```diff
@@ -93,6 +92,57 @@ Your `.eslintrc.{js,yml,json}` configuration file will also include the line:
 
 Because of this line, all of the rules marked "(recommended)" on the [rules page](/docs/rules) will be turned on.  Alternatively, you can use configurations that others have created by searching for "eslint-config" on [npmjs.com](https://www.npmjs.com/search?q=eslint-config).  ESLint will not lint your code unless you extend from a shared configuration or explicitly turn rules on in your configuration.
 
+## Global install
+
+It is also possible to install ESLint globally rather than locally (using `npm install eslint --global`). However, this is not recommended, and any plugins or shareable configs that you use must be installed locally in either case.
+
+## Manual set up
```

Comment:
Adjusting caps, but also maybe "Setup" (noun) instead of "Set Up" (verb)? Unsure of the standard we want to follow for this.

```suggestion
## Manual Set Up
```